src/pages/components/callout-quote.mdx

```diff
@@ -23,9 +24,22 @@ The Callout &mdash; quote pattern uses high contrast colors to create a delightf
 
 ![Default](../../images/pattern/callout-quote/callout-quote.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Paragraph | Long copy | Yes   | 1 | LL (200 / 260)|
+| Source (ex. name)  | Short copy | No  | 1 | XS (65 / 85) |
+| Copy 1 (ex. role)  | Short copy | No | 1 | XS (65 / 85) |
+| Copy 2 (ex. company)  | Short copy | No | 1 | XS (65 / 85)   |
+| Quotation mark type | Option list | Yes | - | Double curved (default)<br/>Single curved<br/>Double angle<br/> Single angle<br/> Low high reversed double curved |
```

Comment:
Because it has a default I wouldn't say the mark type is required.

Reply:
Done :)